pipelines/SDM/SDM_BRT_NDVI.json

```diff
@@ -0,0 +1,1011 @@
+      "label": "STAC URL",
+      "description": "URL of the Stac Catalog to pull environmental predictors and water mask from.",
+      "type": "text",
+      "example": "https://stac.geobon.org/",
+      "weight": 5
+    },
+    "pipeline@152": {
+      "label": "Pseudoabsence Buffer",
+      "description": "The minimum distance a PA is allowed to be from a presence in kilometers",
+      "type": "float",
+      "example": 10,
+      "weight": 6
+    },
+    "pipeline@153": {
+      "label": "Max Candidate Pseudoabsences",
+      "description": "The maximum number of candidate pseudoabsences to consider. This speeds up PA generation on large rasters.",
+      "type": "int",
+      "example": 100000,
+      "weight": 7
+    },
+    "pipeline@154": {
+      "label": "Pseudoabsence proportion",
+      "description": "The number of PAs, given by the proportion of the total occurrences to use.",
+      "type": "float",
+      "example": 2.4,
+      "weight": 8
+    },
+    "VegIndexMax>VegIndexMax.yml@164|study_area_polygon": {
+      "description": "Polygon of the study area of interest",
+      "label": "Polygon of study area",
+      "type": "application/geo+json",
+      "weight": 9,
+      "example": null
+    },
+    "VegIndexMax>VegIndexMax.yml@164|start_date": {
+      "description": "Start date for summarizing",
+      "label": "Start date",
+      "type": "text",
+      "example": "2019-05-01",
+      "weight": 10
+    },
+    "VegIndexMax>VegIndexMax.yml@164|end_date": {
+      "description": "End date for which summarizing NDVI.",
+      "label": "End date",
+      "type": "text",
+      "example": "2019-09-30",
+      "weight": 11
+    },
+    "pipeline@163": {
+      "label": "Environmental predictors",
+      "description": "Vector of strings, collection name followed by '|' followed by item id",
+      "type": "text[]",
+      "example": [
+        "chelsa-clim|bio1",
+        "chelsa-clim|bio2"
+      ],
+      "weight": 12
+    },
+    "data>getGBIFObservations>getGBIFObservations.yml@159|min_year": {
+      "description": "Min year observations wanted",
+      "label": "minimum year",
+      "type": "int",
+      "example": 2010,
+      "weight": 13
+    },
+    "data>getGBIFObservations>getGBIFObservations.yml@159|max_year": {
+      "description": "Max year observations wanted",
+      "label": "maximum year",
+      "type": "int",
+      "example": 2024,
+      "weight": 14
+    }
+  },
+  "outputs": {
+    "pipeline@121|default_output": {
+      "label": "Species",
+      "description": "Name of species",
+      "example": [
+        "Acer saccharum"
+      ],
+      "type": "text[]",
+      "weight": 0
+    },
+    "filtering>cleanCoordinates.yml@34|clean_presence": {
+      "description": "Occurrences from GBIF after cleaning",
+      "label": "Presences",
+      "type": "text/tab-separated-values",
+      "weight": 1
+    },
+    "SDM>BRT>fitBRT.yml@132|pseudoabsences": {
+      "description": "pseudoabsence coordinates",
```

Comment:
Need to explain a bit more what a pseudoabsence is, this will go on the viewer for demonstration purpose, we cannot assume everyone looking at this will be ecologists.